recipe/meta.yaml

```diff
@@ -33,7 +34,7 @@ test:
     - pytest-cache >=1.0
```

Comment:
Since not running the tests, no need to add pytest. 
pip check would be welcome.

Reply:
done!